### PR TITLE
AprilTag Documentation

### PR DESCRIPTION
**Pull request by mdurrani808**

### CODE REVIEW

**Comment by mdurrani808** [comment]
Closes #191

**Comment by mdurrani808** [comment]
Closes #192

**Comment by mdurrani808** [comment]
Close #191 and #192

File: source/docs/getting-started/pipeline-tuning/apriltag-tuning.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+|
+
+Target Family
+-------------
+Target families are defined by two numbers (before and after the h). The first number is the number of bits the tag is able to encode (which means more tags are available in the respective family) and the second is the hamming distance. Hamming distance describes the ability for error correction while identifying tag ids. A high hamming distance generally means that it will be easier for a tag to be identified even if there are errors. However, as hamming distance increases, the number of available tags decreases. The 2023 FRC game will be using 16h5 tags, which can be found `here <https://github.com/AprilRobotics/apriltag-imgs/tree/master/tag16h5>`_. PhotonVision also supports the usage of 36h11 tags.
+
+Decimate
+--------
+
+Decimation (also known as down-sampling) is the process of reducing the sampling frequency of a signal (in our case, the image). Increasing decimate will lead to an increased detection rate while decreasing detection distance. We recommend keeping this at the default value.
```
Comment (gerth2) [inline source/docs/getting-started/pipeline-tuning/apriltag-tuning.rst:17]: > Increasing decimate will lead to an increased detection rate while decreasing detection distance.

Suggest "Increasing Decimate will lead to faster image processing, but requires tags be closer to the camera to be properly identified".

**Review by mcm001 — APPROVED** [review]

**Review by gerth2 — APPROVED** [review]

**Pull request merged**